Automattic/vip-cli
# SQL Import: Fix help prompts for imports and S-R

## Description

- Add examples for the SQL import command
- Add examples for standalone Search and Replace command
- Expose the `output` flag for S-R when it's run during an import (we're currently only exposing it for standalone S-R commands)
- Fix wording of descriptions to be more user-friendly + add warning about `no space` for S-R params

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `VIP_PROXY="<proxy>" node ./dist/bin/vip-import-sql -h`
1. Verify help output looks good:
<img width="1182" alt="help examples" src="https://user-images.githubusercontent.com/29221840/104987204-fba31680-59c9-11eb-91c7-60a72e3034de.png">

5. Run `VIP_PROXY="<proxy>" node ./dist/bin/vip-search-replace -h` and verify help output looks good
